.circleci/config.yml

```diff
@@ -69,18 +69,6 @@ jobs:
       - run:
           name: Install target toolchain
           command: rustup toolchain install stable-x86_64-pc-windows-msvc
-      - run:
-          name: Install Clippy
-          command: rustup component add clippy
-      - run:
-          name: Run Clippy
-          # Remove the first two lines of gitconfig as work around
-          command: |
-            (gc ..\.gitconfig | select -Skip 2) | sc ..\.gitconfig
```

Comment:
Removing this is causing the windows pipeline to fail. It looks like our solution is kinda hacky but it works, it is related to a known [issue](https://github.com/rust-lang/cargo/issues/2078) with CIrcleCI and Cargo.

Maybe a more convenient solution would be to add `- run: git config --global --unset url.ssh://git@github.com.insteadof` but haven't tried it.

Reply:
Cool, thanks for that link. Trying the workaround posted in it first.